.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```

Comment:
Can you explain the reason to change the version number of ubuntu?

Reply:
with the latest ubuntu test are failing with some errors
```
E                 -                'tables': [{'columns': [{'data_type': 'text',
E                 ?                                                       ^^^^
E                 +                'tables': [{'columns': [{'data_type': 'TEXT',
```

This needs to be validated, But for now to limiting ubuntu works. 

[#84-build](https://github.com/tokern/piicatcher/actions/runs/3656227930/jobs/6178423203) there with the error.